2-mandatory/5-reading-list.js

```diff
@@ -16,12 +16,35 @@ In this style of testing it is typical to write out as strings exactly what you
 without using any variables or any logic like loops, template strings or if statements.
 */
 
-const books = [];
+const books = [
+  {
+    title: "JavaScrip",
+    author: "Khadar",
+    alreadyRead: true,
+  },
+  {
+    title: "HTML",
+    author: "Adam",
+    alreadyRead: false,
+  },
+  {
+    title: "CSS",
+    author: "Rayaan",
+    alreadyRead: true,
+  },
+  {
+    title: "REACT",
+    author: "Rahiima",
+    alreadyRead: false,
+  },
+];
   
 // exercise 1
-function logBooks() {
+function logBooks(shelftedBooks) {
```

Comment:
This code could have worked but there's something wrong with the list of aurguements here. Can you see what it is?